### PR TITLE
zlib-devel must be installed for ui tests

### PR DESCRIPTION
**Pull request by vmoravec**
Needed by `nokogiri` gem.

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -4646,7 +4646,7 @@ function onadmin_run_cct()
         ensure_packages_installed git-core gcc make ruby2.1-devel
 
         if [[ $cct_tests =~ ":ui:" ]]; then
-            ensure_packages_installed libqt4-devel libQtWebKit-devel xorg-x11-server xorg-x11-server-extra
+            ensure_packages_installed libqt4-devel libQtWebKit-devel xorg-x11-server xorg-x11-server-extra zlib-devel
```
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: Is the gem compiled in site?

**Comment by gosipyan** [comment]
+1

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (vmoravec) [inline scripts/qa_crowbarsetup.sh]: yes, all cct deps are compiled on the fly into `/vendor` dir

**Comment by aplanas** [comment]
+1

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: Makes sense to create a package?

Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: like this one? https://build.opensuse.org/package/show/devel:languages:ruby:extensions/rubygem-nokogiri

Comment (vmoravec) [inline scripts/qa_crowbarsetup.sh]: @aplanas @bmwiedemann Probably yes, or we could change the configuration in https://github.com/SUSE-Cloud/cct/blob/master/.bundle/config first to accept the gems installed via zypper .  I will look into packaging cct later.

**Pull request merged**